hazelcast/src/test/java/com/hazelcast/map/nearcache/NearCacheTest.java

```diff
@@ -117,11 +117,18 @@ public void testBasicUsage() {
         }
 
         map.clear();
-        for (HazelcastInstance instance : instances) {
-            NearCache nearCache = getNearCache(mapName, instance);
-            int size = nearCache.size();
-            assertEquals("Near Cache size should be 0 but was " + size, 0, size);
-        }
+
+        assertTrueEventually(new AssertTask() {
```

Comment:
i see your point, it is ok to have it on release notes, will work on it with @Serdaro